src/components/FeedbackWidget.js

```diff
@@ -0,0 +1,278 @@
+
+const FixedDot = styled.div`
+  width: 3rem;
+  aspect-ratio: 1;
+  border-radius: 50%;
+  background-color: ${({ theme }) => theme.colors.primary};
+  box-shadow: 0px 4px 4px ${({ theme }) => theme.colors.tableItemBoxShadow};
+  position: fixed;
+  bottom: 1rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    bottom: ${({ bottomOffset }) => 1 + bottomOffset}rem;
+  }
+  right: 1rem;
+  z-index: 98; /* Below the mobile menu */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.1);
+    transition: transform 0.2s ease-in-out;
+  }
+  transition: transform 0.2s ease-in-out;
+`
+
+const ModalBackground = styled.div`
+  display: ${({ isOpen }) => (isOpen ? "block" : "none")};
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  z-index: 1001; /* Above the nav bar */
+`
+
+const Container = styled.div`
+  display: flex;
+  box-sizing: border-box;
+  width: 300px;
+  background-color: ${({ theme }) => theme.colors.ednBackground};
+  border: 1px solid ${({ theme }) => theme.colors.buttonColor};
+  box-shadow: 0px 4px 4px ${({ theme }) => theme.colors.tableItemBoxShadow};
+  border-radius: 0.25rem;
+  position: fixed;
+  right: 2rem;
+  bottom: 5rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    bottom: ${({ bottomOffset }) => 5 + bottomOffset}rem;
+  }
+  z-index: 1002; /* Above the ModalBackground */
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: auto;
+    left: 1rem;
+    right: 1rem;
+  }
+
+  &:hover {
+    transform: scale(1.02);
+    transition: transform 0.2s ease-in-out;
+  }
+  transition: transform 0.2s ease-in-out;
+
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 2rem;
+
+  .title {
+    font-weight: 700;
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+  }
+
+  .subtitle {
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+
+  .timing {
+    font-weight: 700;
+    font-size: 0.75rem;
+    line-height: 1rem;
+    letter-spacing: 0.025em;
+    color: ${({ theme }) => theme.colors.searchBorder};
+  }
+
+  a {
+    width: 100%;
+    color: ${({ theme }) => theme.colors.text} !important;
+  }
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  gap: 1.25rem;
+  width: 100%;
+  * {
+    flex: 1;
+    color: ${({ theme }) => theme.colors.white};
+    font-weight: 700;
+  }
+`
+
+const StyledFeedbackGlyph = styled(FeedbackGlyph)`
+  path {
+    fill: ${({ theme }) => theme.colors.white};
+  }
+`
+
+const IconContainer = styled.div`
+  position: absolute;
+  right: 0.5rem;
+  top: 0.5rem;
+  cursor: pointer;
+  &:hover {
+    transform: scale(1.1);
+    transition: transform 0.2s ease-in-out;
+  }
+  transition: transform 0.2s ease-in-out;
+`
+
+const FeedbackWidget = ({ prompt, className }) => {
```

Comment:
I don't see `prompt` used anywhere. Is this for future-proofing in case we want to have a custom prompt? If not, I think we can remove it for now.

Reply:
Agreed if it's not used we should remove.